Projects/Domain/StudentDomain/Interface/DTO/Request/CreatePortfolioLinkRequestDTO.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+public struct CreatePortfolioLinkRequestDTO: Encodable {
+    public let studentId: String
+    public let periodDay: Int
+
+    public init(
+        studentId: String,
+        periodDay: Int
+    ) {
+        self.studentId = studentId
+        self.periodDay = periodDay
```

Comment:
```suggestion
    public let studentID: String
    public let periodDay: Int

    public init(
        studentID: String,
        periodDay: Int
    ) {
        self.studentID = studentID
        self.periodDay = periodDay
```
스타일 가이드에 맞게 Id 보다 ID로 하는 게 더 좋아보여요!!